common/static/common/js/discussion/discussion_router.js

```diff
@@ -18,9 +18,20 @@
             return child;
         };
 
+    function getSingleThreadRoute(commentable_id, thread_id) {
+        return commentable_id + "/threads/" + thread_id;
+    }
+
     if (typeof Backbone !== "undefined" && Backbone !== null) {
         this.DiscussionRouter = (function(_super) {
 
+            var allThreadsRoute = "",
+                singleThreadRoute = getSingleThreadRoute(":forum_name", ":thread_id"), // :forum_name/threads/:thread_id
+                routes = {};
+
+            routes[allThreadsRoute] = "allThreads";
```

Comment:
Prefer using dot notation to set object properties when the property name allows it (as these do), e.g. `routes.allThreadsRoute`